rollbar/resource_notification_test.go

```diff
@@ -165,6 +165,41 @@ func (s *AccSuite) TestNotificationUpdate() {
 	})
 }
 
+func (s *AccSuite) TestNotificationCreateSpecialEmail() {
+	notificationResourceName := "rollbar_notification.email_notification"
+	// language=hcl
+	config := `
+		resource "rollbar_notification" "email_notification" {
+  rule  {
+
+   trigger = "daily_summary"
+  }
+  channel = "email"
+  config  {
+	summary_time = 2
+	min_item_level = "critical"
+  }
+}
+	`
+	resource.ParallelTest(s.T(), resource.TestCase{
+		PreCheck:     func() { s.preCheck() },
+		Providers:    s.providers,
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					s.checkResourceStateSanity(notificationResourceName),
+					resource.TestCheckResourceAttr(notificationResourceName, "channel", "email"),
+					resource.TestCheckResourceAttr(notificationResourceName, "rule.0.trigger", "daily_summary"),
+					resource.TestCheckResourceAttr(notificationResourceName, "config.0.summary_time", "2"),
+					resource.TestCheckResourceAttr(notificationResourceName, "config.0.min_item_level", "critical"),
```

Comment:
Should we also include defs and tests for each of the email daily summary-related fields?

Reply:
done